kobsio/kobs
# [Jaeger] Plugin jaeger chart red errors

Jaeger Plugin: Change color of erroneous traces in the chart to red

This PR is based on 
-  https://github.com/kobsio/kobs/pull/115
- https://github.com/kobsio/kobs/pull/116


- [x] I added a [CHANGELOG](https://github.com/kobsio/kobs/blob/master/CHANGELOG.md) entry for this change.
- [ ] I adjusted the corresponding [documentation](https://github.com/kobsio/kobs/tree/main/docs) for this change.
- [ ] I adjusted the [values.yaml](https://github.com/kobsio/kobs/blob/main/deploy/helm/kobs/values.yaml) file and the [documentation for all values](https://github.com/kobsio/kobs/blob/main/docs/installation/helm.md).

### How to test:

Run "demo" setup https://kobs.io/installation/demo/

Build local image and push to "local" registry
```bash
docker build -t localhost:5000/kobs:dev -f cmd/kobs/Dockerfile .
docker push localhost:5000/kobs:dev
```

Change to "dev" mode: `kustomize build deploy/demo/kobs/dev | kubectl apply -f -`

Call the bookinfo: 
```
while :; do curl -s "http://bookinfo.demo/productpage" | grep -o "<title>.*</title>"; sleep 1; done
```
Delete the details service from `bookinfo`

`kubectl delete deployment -n bookinfo details-v1`

Open http://kobs.demo/jaeger and check for errors. The graph should show red dots
